.drone.yml

```diff
@@ -15,23 +10,30 @@ platform:
 steps:
 - name: build
   pull: if-not-exists
-  image: docker:19.03.12-dind
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
   commands:
   # wait for docker service to be up before running docker build
-  - n=0; while [ "$n" -lt 60 ] && [ ! -e /var/run/docker.sock ]; do n=$(( n + 1 )); sleep 1; done
+  - n=0; while [ "$n" -lt 60 ] && ! docker stats --no-stream; do n=$(( n + 1 )); sleep 1; done
+  - if ! docker stats --no-stream; then echo "Docker appears not to be running"; exit 1; fi
   - docker build -t alpine:$${DRONE_COMMIT_SHA} . --no-cache
-  volumes:
-  - name: dockersock
-    path: /var/run
+  environment:
+    DOCKER_HOST:
```

Comment:
You can remove this env var when using the ACP dind image

Reply:
Good catch - copied this from pipeline using `quay.digital.homeoffice.gov.uk/ukhomeofficedigital/dind-awscli:19.03.12-dind-1.18.55` as the Drone step's base image, which *does* need the extra env var.

Reply:
Oh I wasn't aware of that, must have been an initial image before it was simplified.